portfolio/src/main/webapp/index.html

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
+    <div id="content" id= "border">
```

Comment:
You should only have one id attribute per element. You can have multiple classes associated with an element though